### ISSUE TITLE
The requirement for cpu memory has not indicated

### ISSUE THREAD

**Issue by Sarah20187** [amitshaked/resmatch#3]
Could you please add the instrument for that?

**Comment by amitshaked** [comment]
What did you try to run? What limit did you encounter?

**Comment by Sarah20187** [comment]
While training the Dispnet, I found that you load all the data into the memory. As I just have 32GB memory and it is not enough to load all the data. Then I try restore the cost volume to lmdb, and read it while makepatch to minibatch, then reading cost volume costs too much time which makes training take long time. So could you tell me what's your hardware configuration?

**Comment by amitshaked** [comment]
Yeah you're right, I use 64G ram card. Probably the best thing to do would be to load the data online as you suggested, but in separate thread to make it faster. Would be great if you could do it and pull request. 
You can also do some hack and train on half the data each time (make sure you chose it randomly).

**Comment by Sarah20187** [comment]
And I remember that you use roc to evaluate the predicted confidence, while
in my view that smaller roc means better result. So I'm curious about this
problem. Could you show me your equations of how to calculate ROC?

**Comment by amitshaked** [comment]
I simply calculate the area under the curve (the bigger the better) of the classifier. Please see section 4.2 in the paper for further details.

**Comment by Sarah20187** [comment]
@amitshaked sorry for making mistakes. I mean for auc, the smaller, the better.(see `Figure 6` of paper Leveraging Stereo Matching with Learning-based Confidence Measures)

**Comment by Junjiewu2279** [comment]
@zhFuECL Hi, I have the same problem(out of memory). As I just have 8GB memory and it is not enough to load all the data. Can you tell me your solution(details)? Thanks!

**Comment by Junjiewu2279** [comment]
@zhFuECL Where should I modify the code?
